templates/header.hbs

```diff
@@ -16,7 +16,7 @@
           data-label-3="Student Money"
           data-target-3="_blank"
         ></div>
-        <a href="{{settings.base_url}}/help/">Help</a>
+        <a href="https://help.futurefinance.com/hc/en-us">Help</a>
```

Comment:
should this link have `active` class to make it highlighted?

Reply:
Thought so, but then wondered if that link should be highlighted if they browse to https://help.futurefinance.com/hc/en-us/categories/360002179260-Guarantors ?

Reply:
I guess I will add class=active

Reply:
Done